src/lib/Behat/PageElement/UniversalDiscoveryWidget.php

```diff
@@ -14,8 +14,8 @@ class UniversalDiscoveryWidget extends Element
 {
     public const ELEMENT_NAME = 'UDW';
     private const UDW_TIMEOUT = 20;
-    private const UDW_DISAPPEAR_TIMEOUT = 1;
-    private const UDW_BRANCH_LOADING_TIMEOUT = 5;
+    private const UDW_DISAPPEAR_TIMEOUT = 2;
+    private const UDW_BRANCH_LOADING_TIMEOUT = 10;
```

Comment:
Is it really necessary? The issue is that selecting an item from a single select UDW will take at least 10 seconds every time (`isMultiSelect` call is blocking for UDW_BRANCH_LOADING_TIMEOUT seconds when UDW is single-select).

Maybe `isMultiSelect` can be changed so that UDW mode is detected in a better way (as far as I remember it wasn't possible when it was implemented).